rt/features/logging/src/main/java/org/apache/cxf/ext/logging/MaskSensitiveHelper.java

```diff
@@ -27,8 +27,8 @@
 
 public class MaskSensitiveHelper {
     private static final String ELEMENT_NAME_TEMPLATE = "-ELEMENT_NAME-";
-    private static final String MATCH_PATTERN_XML_TEMPLATE = "(<-ELEMENT_NAME-.*?>)(.*?)(</-ELEMENT_NAME->)";
-    private static final String REPLACEMENT_XML_TEMPLATE = "$1XXX$3";
+    private static final String MATCH_PATTERN_XML_TEMPLATE = "(<(\\w+:)?-ELEMENT_NAME-.*?>)(.*?)(</(\\w+:)?-ELEMENT_NAME->)";
```

Comment:
@gpitteloud thanks for the pull request, I think the characters that could be included into XML namespace name are wider than Java's `\w` character class.

[1] https://www.w3.org/TR/xml11/#NT-NameChar

Reply:
@reta thanks for pointing this. I will update the MR to add all extra chars.

Reply:
sorry for the formatting changes, I didn't notice my IDE changed that

Reply:
not a problem, thank you